website/docs/core-concepts/components/remote-state-backend.md

```diff
@@ -0,0 +1,321 @@
+it requires - this is the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege).
+
+We can add the `remote_state_backend` and `remote_state_backend_type` to override the required attributes from the
+`backend` section:
+
+```yaml title="stacks/orgs/acme/_defaults.yaml"
+terraform:
+  backend_type: s3  # s3, remote, vault, azurerm, gcs, cloud
+  backend:
+    s3:
+      acl: "bucket-owner-full-control"
+      encrypt: true
+      bucket: "your-s3-bucket-name"
+      dynamodb_table: "your-dynamodb-table-name"
+      key: "terraform.tfstate"
+      region: "your-aws-region"
+      role_arn: "arn:aws:iam::xxxxxxxx:role/terraform-backend-read-write"
+
+  remote_state_backend_type: s3 # s3, remote, vault, azurerm, gcs, cloud, static
+  remote_state_backend:
+    s3:
+      role_arn: "arn:aws:iam::xxxxxxxx:role/terraform-backend-read-only"
+      # Override the other attributes from the `backend.s3` section as needed
+```
+
+<br/>
+
+In the example above, we've overridden the `role_arn` attribute for the `s3` backend to use the read-only role when
+accessing the remote state of all components. All other attributes will be taken from the `backend` section (Atmos
+deep-merges the `remote_state_backend` section with the `backend` section).
+
+When working with Terraform backends and writing/updating the state, the `terraform-backend-read-write` role will be
+used. But when reading the remote state of components, the `terraform-backend-read-only` role will be used.
+
+## Brownfield Development
+
+[Brownfield development](https://en.wikipedia.org/wiki/Brownfield_(software_development)) is a term commonly used in the
+information technology industry to describe problem spaces needing the development and deployment of new software
+systems in the immediate presence of existing (legacy) software applications/systems. This implies that any new software
+architecture must take into account and coexist with the existing software. The term "brownfield" itself is borrowed
+from urban planning, where it describes the process of developing on previously used land that may require cleanup or
+modification.
+
+In the context of DevOps, brownfield development involves integrating new tools, practices, and technologies into
+established systems. This can be challenging due to several factors:
+
+- **Legacy Systems**: These are older technologies or systems that are still in use. They might not support modern
+  practices or tools easily, and modifying them can be risky and time-consuming.
+
+- **Complex Integrations**: Existing systems often have a complex set of integrations and dependencies which need to be
+  understood and managed when new elements are introduced.
+
+- **Cultural Shifts**: DevOps emphasizes a culture of collaboration between development and operations teams. In
+  brownfield projects, shifting the organizational culture can be a significant hurdle as existing processes and
+  mindsets may be deeply ingrained.
```

Comment:
Let's remove this one as I don't think it adds anything helpful as it relates to IaC and what we want to solve with static backends.

```suggestion
```